core/src/test/java/io/confluent/kafka/schemaregistry/SSLClusterTestHarness.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2014 Confluent Inc.
```

Comment:
I wasn't sure if we keep the founding year of Confluent, or this year. I'll change to this year.